bootcamp_module08/core/tests/test_student_03.py

```diff
@@ -26,3 +27,19 @@ def test_count_substring_none():
     expected_count = 0
     observed_count = count_substring(test_string, test_substring)
     assert expected_count == observed_count
+
+
+def test_count_substring_lowercase():
+    test_string = "AGTCTAGCatgctATG"
+    test_substring = "AtG"
+
+    expected_count = 2
+    observed_count = count_substring(test_string, test_substring)
+    assert expected_count == observed_count
+
+
+def test_reverse_string():
```

Comment:
This really belongs in `bootcamp_module08/contrib/tests`. Can you move it there?